CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Each version section may have have subsections for: _Added_, _Changed_, _Removed
 - Make sure the default bucket is large enough with word based batching when the source is longer than the target (Previously
 there was an edge case where the memory usage was sub-optimal with word based batching and longer source than target sentences).
 
+### Added
+- ROUGE score evaluation. It can be used as the stopping criterion for tasks such as summarization.
+
```

Comment:
Since this affect Sockeye itself, you should increment the version number (in `__init__.py`) and add that information here.